src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamProviderFactory.scala

```diff
@@ -42,6 +43,12 @@ object StreamProviderFactory {
             new File(settings.progressDir, Constants.EventHubProgressDir).getPath)
         )
       )
+      .withFactories(
+        List(
```

Comment:
this is going to mean that @Smarker will  have to add a record into the `streams` table upon a called to `CreateOrReplaceSite`, as long as the eventhub environment is properly setup.